sdk/template/Azure.Template/src/Azure.Template.csproj

```diff
@@ -1,22 +1,23 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>This is a template project to demonstrate how to create a package that uses code generation as well as use for testing our build and release pipelines</Description>
-    <AssemblyTitle>Azure SDK Template</AssemblyTitle>
-    <Version>1.0.3-beta.19</Version>
+    <Description>This is the Template client library for developing .NET applications with rich experience.</Description>
+    <AssemblyTitle>Azure SDK Code Generation Template for Azure Data Plane</AssemblyTitle>
+    <Version>1.0.0-beta.1</Version>
     <PackageTags>Azure Template</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>$(NoWarn);419;AZC0012</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants>$(DefineConstants);EXPERIMENTAL</DefineConstants>
     <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Azure.Core" />
-    <PackageReference Include="System.Text.Json" />
```

Comment:
Why remove the reference to System.Text.Json?  I expect most libraries will need it, and it will help clarify that this is the Azure SDK supported library for Json.